util/test_utils/test_configurations.py

```diff
@@ -6,6 +6,121 @@ class ConfigurationTests(unittest.TestCase):
     def setUp(self):
         self.config_utils = ConfigurationUtils("")
 
+    def test_GIVEN_component_xml_WHEN_parsed_THEN_can_extract_a_single_component(self):
+        xml = """<?xml version="1.0" ?>
```

Comment:
What do you mean? Also I did not make any changes to the component xml tests, maybe it shows up that I changed because I reformatted with different indenting?

Reply:
I mean that the xml in this test and the one in the test below is basically the same with minor tweaks. Therefore it should be put into a common place that each test calls. If you didn't change these tests then there's no need to do it, though it would improve the code, but you should do it for your block tests.